src/Models/UserIdentityModel.php

```diff
@@ -329,13 +329,48 @@ public function revokeAllAccessTokens(User $user): void
         $this->checkQueryReturn($return);
     }
 
+    /**
+     * Force password reset for multiple users.
+     *
+     * @param int[]|string[] $userIds
+     *
+     * @return mixed
+     */
+    public function forceMultiplePasswordReset(array $userIds)
+    {
+        $this->where(['type' => Session::ID_TYPE_EMAIL_PASSWORD, 'force_reset' => 0]);
+        $this->whereIn('user_id', $userIds);
+        $this->set('force_reset', 1);
+
+        return $this->update();
+    }
```

Comment:
We return void (and throws an exception) when the method is a *command*, not a *query*.

```suggestion
    public function forceMultiplePasswordReset(array $userIds): void
    {
        $this->where(['type' => Session::ID_TYPE_EMAIL_PASSWORD, 'force_reset' => 0]);
        $this->whereIn('user_id', $userIds);
        $this->set('force_reset', 1);
        $return = $this->update();

        $this->checkQueryReturn($return);
    }
```